test/firtool/lower-layers.fir

```diff
@@ -316,3 +316,22 @@ circuit Foo:
 
 ; CHECK: FILE "B{{[/\]}}layers-Foo-B.sv"
 ; CHECK: `include "layers-Bar-B.sv"
+
+; // -----
+
+; Check that bindfile for Top should include layers-Bar-A, not layers-Foo-A,
+; respecting the defname of an extmodule.
+
+FIRRTL version 5.1.0
+circuit Top:
+  layer A, bind:
+
+  extmodule Foo knownlayer A:
+    defname = Bar
+
+  public module Top:
+    inst foo of Foo
+
+; CHECK:      FILE "layers-Top-A.sv"
+; CHECK:      `include "layers-Bar-A.sv"
+; CHECK-NOT:  `include "layers-Foo-A.sv"
```

Comment:
Nit: This could technically pass if the `` `include `` for Foo showed up before the `` `include `` for Bar. Consider:


```suggestion
; CHECK-NOT:  `include
; CHECK:      `include "layers-Bar-A.sv"
; CHECK-NOT:  `include
```

Reply:
I dropped the check-not, honestly it wasn't really functional, it was only "explanatory", if that makes sense.